src/App.vue

```diff
@@ -5,6 +5,7 @@
     <mapbox-map
       slot="map"
       :access-token="accessToken"
+      :padding="{ left: appNavigationOpen ? appNavigationWidth : 0 }"
```

Comment:
Maybe move this logic to a computed prop like `mapLeftPadding` and keep as much direct logic out the template.